fpdf/enums.py

```diff
@@ -199,6 +199,26 @@ def coerce(cls, value):
         return super(cls, cls).coerce(value)
 
 
+class VAlign(CoerciveEnum):
+    """Defines how to vertically render text in a cell.
+    Default value is CENTER"""
+
+    C = intern("CENTER")
```

Comment:
in CSS, the vertical one is called "middle" to distinguish it from the horizontal "center". Maybe we should follow this pattern?

Reply:
Yes, I agree. Updated and pushed.